js/src/views/Account/account.js

```diff
@@ -80,6 +82,15 @@ class Account extends Component {
     if (prevAddress !== nextAddress) {
       this.setVisibleAccounts(nextProps);
     }
+
+    const { isTestnet } = nextProps;
+    if (typeof isTestnet === 'boolean' && !this.state.verificationStore) {
+      const { api } = this.context;
+      const { address } = nextProps.params;
+      this.setState({
+        verificationStore: new VerificationStore(api, address, isTestnet)
```

Comment:
Sorry, thought you were talking about the badges. Just putting the `store` as a class prop didn't come to my mind, will move.